src/openforms/submissions/templates/report/submission_report.html

```diff
@@ -86,7 +86,10 @@ <h2 class="subtitle">{{ submission_step_node.render }}</h2>
                                                 <div aria-labelledby="{{ node_path }}" class="submission-step-row__value{% if not component_node.display_value %} submission-step-row__value--empty{% endif %}">
                                                     {% if component_node.display_value %}
                                                         {{ component_node.display_value }}
-                                                    {% elif not component_node.layout_modifier %}
+                                                        {% comment %} 
+                                                            the component_label is the custom name that we have for the family members components
+                                                        {% endcomment %}
+                                                    {% elif not component_node.layout_modifier and component_node.key != "component_label" %}
```

Comment:
I don't understand what's going on here - is this still relevant now that a `layout_modifier` is set?

Reply:
Yes, but I understood that this is only relevant for the parent and not for each item like it is for the `editgrid` for example. The way that the ` editgrid` is implemented is different so that's why I had to add this statement here.

Reply:
Hrm, I don't like how much specificity is leaking into the templates that are meant to be generic node visitors. Is there no other way we can express this? E.g. for the edit grid we have `EditGridNode` which yields `EditGridGroupNode` from its children - can't we do something similar there?

We would then be able to piggy-back on the `layout_modifier: str = "editgrid-group"` which gives us the right styling "for free".

This applies for both the children and partners components I think.